apps/console/src/features/identity-providers/components/forms/authenticators/google-authenticator-form.tsx

```diff
@@ -184,24 +201,55 @@ export const GoogleAuthenticatorForm: FunctionComponent<GoogleAuthenticatorFormP
             const meta: CommonAuthenticatorFormFieldMetaInterface = metadata?.properties
                 .find((meta) => meta.key === value.key);
 
+            /**
+            * Parsing string  to boolean only for Google One Tap value
+            */
+            let localValue : any;
+
+            if (value.key === IdentityProviderManagementConstants.GOOGLE_ONE_TAP_ENABLED) {
+                if (value.value === "true") {
+                    localValue = true;
+                } else {
+                    localValue = false;
+                }
+            } else {
+                localValue = value.value;
+            }
+
             resolvedFormFields = {
                 ...resolvedFormFields,
                 [ value.key ]: {
                     meta,
-                    value: value.value
+                    value: localValue
                 }
             };
 
             resolvedInitialValues = {
                 ...resolvedInitialValues,
-                [ value.key ]: value.value
+                [ value.key ]: localValue
             };
         });
 
         setFormFields(resolvedFormFields);
         setInitialValues(resolvedInitialValues);
     }, [ originalInitialValues ]);
 
+    /**
+     * Checking the current user is a super user or not.
```

Comment:
Add newline after the method description.
```suggestion
     * Checking the current user is a super user or not.
     
```

Reply:
Fixed 4c8b96d715217e1abdecfa56cb821a50386ac52b